core-plugins/docs/FTPDeprecated-batchsource.md

```diff
@@ -1,5 +1,6 @@
 # FTP Batch Source
```

Comment:
we should also make this deprecated in case font color is changed in future

Reply:
fixed